book_source/02_demos_tutorials_workflows/05_developer_workflows/02_git/01_using-git.Rmd

```diff
@@ -294,19 +265,7 @@ The Rstudio documentation includes useful overviews of [version control](http://
 
 Once you have git installed on your computer (see the [Rstudio version control](http://www.rstudio.com/ide/docs/version_control/overview) documentation for instructions), you can use the following steps to install the PEcAn source code in Rstudio.
 
-#### Creating a Read-only version:
-
-This is a fast way to clone the repository that does not support contributing new changes (this can be done with further modification). 
-
-1.  install Rstudio (www.rstudio.com)
-2.  click (upper right) project
-*   create project
-*   version control
-*   Git - clone a project from a Git Repository
-*   paste https://www.github.com/PecanProject/pecan
-*   choose working dir. for repo
-
-#### For development:
+##### Changes without requiring a username and password:
```

Comment:
You'll need a username whether forking or not -- would it be clearer to say "without membership in the PEcAn organization"? or similar?

Reply:
I'll update this to indicate that the forking is the preferred method